test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -4028,5 +4030,1474 @@ items:
     - get
     - patch
     - update
+- apiVersion: v1
+  kind: ClusterRole
+  metadata:
+    annotations:
+      rbac.authorization.kubernetes.io/autoupdate: "true"
```

Comment:
> Does origin do anything with this?

It will clone it back during the later sync: https://github.com/openshift/origin/pull/14064